src/main/resources/assets/additionalredstone/lang/zh_cn.json

```diff
@@ -0,0 +1,33 @@
+{
+	"itemGroup.additionalredstone.main" : "更多红石元件",

```

Comment:
Can you reverse this line to `Additional Redstone`, I like to keep the item-group name consistent between all languages

Reply:
if you want, yes.
"additional redstone" is not hard to understand, so keeping the English name is ok